### PR TITLE
Add mac install tips

### PR DESCRIPTION
**Pull request by inderpreet99**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,23 @@
-# ShabadOS-Core
+# ShabadOS-Desktop
```
Comment (bhajneet) [inline README.md]:
```suggestion
# Shabad OS Desktop
```

File: README.md [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
 
 Contains the standalone frontend and backend applications for ShabadOS, with an Electron wrapper.
 
 ## Development
 
 The `app` folder contains the Electron wrapper, the backend Node application, and the frontend React application.
 
-To install dependencies, run `npm install`, which will also install the dependencies for the frontend and backend. 
+To install dependencies, run `npm install`, which will also install the dependencies for the frontend and backend.
```
Comment (bhajneet) [inline README.md]:
```suggestion
Requirements:
* To use docker on Windows you will need the Pro or higher version.
* To use npm on macOS you will need Xcode and the latest npm via `code` and `code`. (Note about node-gyp in parentheticals).

To install dependencies, run `npm install`, which will also install the dependencies for the frontend and backend.
```

File: README.md [hunk 3]
```diff
@@ -1,17 +1,23 @@
...
 
 To spin up the development servers, run `npm run dev`, which will launch a frontend server on `https://localhost:3000`, and a backend server on `https://localhost:42425`.
 
 If you'd prefer to use docker-compose, `docker-compose up` will install the dependencies and spin up the backend and frontend.
 
+## Installation troubleshooting
```
Comment (bhajneet) [inline README.md]: Remove this section

File: README.md (hunk 1 above)
Comment (bhajneet) [reply README.md]: Let's stick with Shabad OS going forward (OS = Open-Source) as it is branded most everywhere else (Social Media, Website, Database repo, etc.).

File: README.md (hunk 2 above)
Comment (inderpreet99) [reply README.md]: How about this instead:

```suggestion
Requirements:
* [Node](https://nodejs.org/en/download/)
* On Windows, you will need the Pro or higher version of Docker.
* On macOS, you will need Xcode Tools via `xcode-select --install` and the latest npm via `npm install --global npm`.

To install dependencies, run `npm install`, which will also install the dependencies for the frontend and backend.
```

Comment (inderpreet99) [reply README.md]: Not sure I understand. Are you saying one can either use Node on the dev machine or docker as a virtual machine? I just took the docker line that you provided above.

Comment (Harjot1Singh) [reply README.md]: It's more that you could launch Shabad OS server in a docker container for dev, I'd probably remove those instructions and only reference npm.

Comment (bhajneet) [reply README.md]: Sounds good to me

**Comment by inderpreet99** [comment]
@Harjot1Singh please merge this.

**Pull request merged**